docs/user-guide/work-packages/work-package-views/README.md

```diff
@@ -49,4 +50,11 @@ Then, the work package with all its details will be displayed.
 
 The arrow next to the subject will bring you back to the list view. 
 
-![back-to-list-view](image-20201217125332403.png)
\ No newline at end of file
+![back-to-list-view](image-20201217125332403.png)
+
+## Card view
+
+You can also display the work package list in a card view. To switch to the card view in the work package list, you must select the card icon at the top right of the work package list. 
+
+<video src="/Users/jankeppler/Desktop/Bildschirmaufnahme 2022-02-11 um 18.11.56.mov"></video>
```

Comment:
This video is only on your computer and it will not work in many browsers, since mov is not supported by many. Please create animated gifs.